server/wfnews-api/wfnews-service-api/src/main/java/ca/bc/gov/nrs/wfnews/service/api/v1/validation/ModelValidator.java

```diff
@@ -194,4 +195,14 @@ public List<Message> validateAttachment(Attachment attachment, long effectiveAsO
 		logger.debug(">validateAttachment " + results.size());
 		return results;
 	}
+
+	public List<Message> validateReport(SituationReport report, long effectiveAsOfMillis) throws DaoException {
+		logger.debug("<validateReport");
+		Class<?>[] groups = new Class<?>[] { AttachmentConstraints.class};
```

Comment:
I guess this should be **SituationReportConstraints.class**?

Reply:
Yes, fixed!